slo_generator/backends/datadog.py

```diff
@@ -113,17 +113,46 @@ def query_slo(self, timestamp, window, slo_config):
         slo_id = slo_config['backend']['measurement']['slo_id']
         from_ts = timestamp - window
         slo_data = self.client.ServiceLevelObjective.get(id=slo_id)
+        LOGGER.debug(
+            f"SLO data: {slo_id} | Result: {pprint.pformat(slo_data)}")
+        try:
+            data = self.client.ServiceLevelObjective.history(id=slo_id,
+                                                             from_ts=from_ts,
+                                                             to_ts=timestamp)
+            LOGGER.debug(
+                f"Timeseries data: {slo_id} | Result: {pprint.pformat(data)}")
+            good_event_count = data['data']['series']['numerator']['sum']
+            valid_event_count = data['data']['series']['denominator']['sum']
+            bad_event_count = valid_event_count - good_event_count
+            return (good_event_count, bad_event_count)
+        except (KeyError) as exception:# monitor-based SLI
+            sli_value = data['data']['overall']['sli_value'] / 100
+            LOGGER.debug(exception)
+            return sli_value
+
+    def query_slo_monitor(self, timestamp, window, slo_config):
```

Comment:
Now that you have merged the two into `query_slo`, we can remove the `query_slo_monitor` method.